serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1,8 +1,10 @@
 = Creating your first Serverless Workflow service
 
+:quarkus_cli_platform_redhat:
```

Comment:
Can you please move it to `antora.yaml` file?

Reply:
Done.